packages/lodestar-cli/src/options/otherOptions.ts

```diff
@@ -0,0 +1,15 @@
+import {Options} from "yargs";
+
+export interface IGenesisStateFileOptions {
+  chain: {
+    genesisStateFile?: string;
+  };
+}
+
+export const genesisStateFileOptions = {
+  "chain.genesisStateFile": {
```

Comment:
why not put this in `packages/lodestar-cli/src/options/beaconNodeOptions/chain.ts`?  what's the reasoning behind putting it in a separate file?

Reply:
genesisStateFile is not actually part of the `BeaconNodeOptions` type. Mixing actual options of `BeaconNodeOptions` with random options scoped only to the CLI is confusing. The entire `options` object is passed to the BeaconNode constructor as if it was `BeaconNodeOptions`, so not merging this option into `beaconNodeOptions` attempts to make this nuance clearer